rest-api/src/main/resources/paths/index.yml

```diff
@@ -353,6 +355,8 @@
     $ref: ./studies/v3_studies_studyId_participants_userId_activities_surveys_surveyGuid.yml
 /v3/studies/{studyId}/participants/{userId}/activities/compoundactivities/{taskId}:
     $ref: ./studies/v3_studies_studyId_participants_userId_activities_compoundactivities_taskId.yml
+/v3/studies/{studyId}/participants/{userId}/requestInfo:
+    $ref: ./studies/v3_studies_studyId_participants_userId_requestInfo.yml
```

Comment:
URLs should be all lowercase... URLs are case sensitive and so mixed casing can be a pain for people. .../requestinfo. It should be the same in BridgeServer2.

Reply:
We already have /v3/participants/{userId}/requestInfo. If we make this lower-case, then it would be inconsistent with the existing one.